search.go

```diff
@@ -268,6 +268,9 @@ func NewSearchRequest(
 	}
 }
 
+// SearchWithPaging augments the given search request with a paging control containing the given paging size, then
+// issues a search with buffered paging using the request. All paged responses are buffered so that the result can
+// be returned atomically.
 func (l *Conn) SearchWithPaging(searchRequest *SearchRequest, pagingSize uint32) (*SearchResult, error) {
```

Comment:
What would I pass through in the third case? `SearchWithPaging(mySearchRequest, someIgnoredInt)` ?